src/extension.ts

```diff
@@ -74,7 +74,7 @@ export function activate(context: vscode.ExtensionContext) {
   context.subscriptions.push(vscode.commands.registerCommand(
     "trivy-vulnerability-scanner.scan",
     () => {
-      const trivyScanCmd = "trivy --quiet filesystem --security-checks config,vuln --exit-code=10";
+      const trivyScanCmd = "trivy --quiet filesystem --scanners config,vuln --exit-code=10";
```

Comment:
I also see this warning:
WARN	'--scanner config' is deprecated. Use '--scanner misconfig' instead. See https://github.com/aquasecurity/trivy/discussions/5586 for the detail.
```suggestion
      const trivyScanCmd = "trivy --quiet filesystem --scanners misconfig,vuln --exit-code=10";
```
This is relatively new, not sure if it should be changed already or if it could cause trouble with older versions (the extension would a version check or the minimal trivy version has to be updated I guess?)